src/arch/xtensa/configs/mt8195_defconfig

```diff
@@ -0,0 +1,17 @@
+CONFIG_MT8195=y
+CONFIG_CORE_COUNT=1
+CONFIG_XT_VIRTUAL_OPS=1
+CONFIG_COMP_VOLUME=y
+CONFIG_COMP_SRC=n
+CONFIG_COMP_FIR=n
+CONFIG_COMP_IIR=n
+CONFIG_COMP_DCBLOCK=n
+CONFIG_COMP_TDFB=n
+CONFIG_COMP_TONE=n
+CONFIG_COMP_MIXER=n
+CONFIG_COMP_MUX=n
+CONFIG_COMP_SWITCH=n
+CONFIG_COMP_KPB=n
+CONFIG_COMP_SEL=n
+CONFIG_COMP_ASRC=n
+
```

Comment:
I wonder if you need to specify all of these components here. Can you please check if the default value isn't already the one you need.